src/main/java/com/example/gildedrose/service/ItemRepository.java

```diff
@@ -0,0 +1,24 @@
+package com.example.gildedrose.service;
+
+import com.example.gildedrose.item.Updateable;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+
+@Service
+public class ItemRepository {
+    private ArrayList<Updateable> items;
```

Comment:
At the risk of being silly - at some point you'll probably want to look up one item at a time. Will each item have an ID? It could just be the index into the Array I suppose.